releaseNotes/Microsoft.Featuremanagement.Telemetry.ApplicationInsights.md

```diff
@@ -2,6 +2,14 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0-preview3 - April 10, 2024
```

Comment:
Did we update this library for the base library reference too?

Reply:
Yes. I will add it.